appveyor.yml

```diff
@@ -10,27 +10,34 @@ install:
   - SET PATH=C:\Ruby%ruby_version%\bin;%PATH%
   - ps: |
       $rv = $env:ruby_version 
-      if ( $rv -lt '24' ) {
-        gem install minitest hoe rake-compiler rubocop:0.68.1 --no-document --conservative
-      } else {
-        gem install minitest hoe rake-compiler rubocop        --no-document --conservative
+      if ($rv -eq '_trunk') {
+        $trunk_uri = 'https://ci.appveyor.com/api/projects/MSP-Greg/ruby-loco/artifacts/ruby_trunk.7z'
```

Comment:
Unfortunately, We never use the 3rd party package without platformer. Because it's hard to know its configuration.

Reply:
Ok.  I guess it's caused a lot of problems in rdoc, reline, & RubyGems (and other external repos)?  Also, who exactly is 'we'?

JFYI:

1. The recent issue in Appveyor (https://ci.appveyor.com/project/ruby/racc/builds/25815248), I think I fixed it, but that's a UI setting that who knows whether Appveyor may reset.  I set my fork UI setting that way, and it passed.

2. If you look here:
https://ci.appveyor.com/project/ruby/racc/builds/25807470/job/tveyvfy72pc7es1s#L269
and
https://travis-ci.org/ruby/racc/jobs/555543268#L500
you'll see that Hoe clobbers Ruby 2.4 rake test.  I locked the Hoe version in Appveyor.yml, whether it should be added to travis, your call.  As mentioned above, it's fixed in Hoe master.

Reply:
You should separate the issues. 

1. Add trunk task. I requested the remove your custom package for trunk.
2. The issues of Hoe

I'm okay to apply your patch for 2.